packages/cli/src/databases/entities/credentials-entity.ts

```diff
@@ -29,6 +29,9 @@ export class CredentialsEntity extends WithTimestampsAndStringId implements ICre
 	@OneToMany('SharedCredentials', 'credentials')
 	shared: SharedCredentials[];
 
+	@Column({ default: false })
+	managed: boolean;
```

Comment:
I'm wondering if a `type` column would make more sense with possible values of `"default" | "managed"` 🤔 I don't see us adding a third one any time soon tho. So I'm fine with this or `type`. Your call. If we keep it as a boolean, we could name it `isManaged` to be explicit that it's a boolean.

Also would be nice to have a comment here explaining what it means that a credential is "managed"

Reply:
My original idea was also call it `type`, but I do not see a future where we need another type 🤔. Let's call it `isManaged` and add the comment.